5.3.lahsheshanit.py

```diff
@@ -0,0 +1,29 @@
+# The function extracts the secret messages from the logo file
+def messages():
```

Comment:
Separate code to small functions- 
https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be

Reply:
I fixed it.